docs/Translation.md

```diff
@@ -130,6 +130,69 @@ const App = () => (
 
 Check [the translation setup documentation](./TranslationSetup.md) for details about `ra-i18n-polyglot` and how to configure it.
 
+## `ra-i18n-i18next`
+
+React-admin also provides a package called `ra-i18n-i18next` that leverages [the i18next library](https://www.i18next.com/) to build an `i18nProvider` based on a dictionary of translations.
+
+You might prefer this package over `ra-i18n-polyglot` when:
+- you already use i18next services such as [locize](https://locize.com/)
+- you want more control on how you organize translations, leveraging [multiple files and namespaces](https://www.i18next.com/principles/namespaces)
+- you want more control on how you [load translations](https://www.i18next.com/how-to/add-or-load-translations)
+- you want to use features not available in Polyglot such as:
+    - [advanced formatting](https://www.i18next.com/translation-function/formatting);
+    - [nested translations](https://www.i18next.com/translation-function/nesting)
+    - [context](https://www.i18next.com/translation-function/context)
+
+```tsx
+// in src/i18nProvider.js
+import { useI18nextProvider, convertRaTranslationsToI18next } from 'ra-i18n-i18next';
+import en from 'ra-language-english';
+import fr from 'ra-language-french';
+
+const i18nInstance = i18n.use(
+    resourcesToBackend(language => {
+        if (language === 'fr') {
+            return import(
+                `ra-language-french`
+            ).then(({ default: messages }) =>
+                convertRaTranslationsToI18next(messages)
+            );
+        }
+        return import(`ra-language-english`).then(({ default: messages }) =>
+            convertRaTranslationsToI18next(messages)
+        );
+    })
+);
+
+export const useMyI18nProvider = useI18nextProvider({
+    i18nInstance,
+    availableLocales: [
+        { locale: 'en', name: 'English' },
+        { locale: 'fr', name: 'French' },
+    ],
+});
+
+// in src/App.js
+import { Admin } from 'react-admin';
+import { useMyI18nProvider } from './i18nProvider';
+
+const App = () => {
+    const i18nProvider = useMyI18nProvider();
```

Comment:
I might be wrong but, looking at the code examples from the storybook, it looks like `useMyI18nProvider` is not callable.

Reply:
You are wrong, we do call it in the stories

Reply:
You're right, this code is completely wrong actually

Reply:
Fixed